backends/aerospike.go

```diff
@@ -109,11 +110,18 @@ func (a *AerospikeBackend) Put(ctx context.Context, key string, value string, tt
 func formatAerospikeError(err error, caller ...string) error {
 	if err != nil {
 		msg := "Aerospike"
+
 		for _, str := range caller {
 			if len(str) > 0 {
 				msg = fmt.Sprintf("%s %s", msg, str)
 			}
 		}
+
+		if aerr, ok := err.(as_types.AerospikeError); ok {
+			if aerr.ResultCode() == as_types.KEY_NOT_FOUND_ERROR {
+				return fmt.Errorf("%s: %s", msg, GetKeyNotFound)
+			}
+		}
```

Comment:
Looks good to me. Can you please verify you tested this end-to-end with a real Aerospike instance?

Reply:
I did